onnxruntime/test/xctest/xcgtest.mm

```diff
@@ -0,0 +1,269 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
```

Comment:
I am not sure if we suppose to add this here

Reply:
There are some modification on this file and I saw there was an similar on the core code.